sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_constants.py

```diff
@@ -57,6 +58,21 @@ class EvaluationRunProperties:
     EVALUATION_SDK = "_azureml.evaluation_sdk_name"
 
 
+class AggregationType(enum.Enum):
```

Comment:
Do we need to add experimental decorator for it?

Reply:
Good idea, adding.